packages/office-ui-fabric-react/src/demo/components/DemoPage.tsx

```diff
@@ -0,0 +1,54 @@
+import { IDemoPageProps } from './DemoPage.types';
+import { ComponentPage, ExampleCard, PropertiesTableSet, PageMarkdown } from '@uifabric/example-app-base';
+import * as React from 'react';
+import { ComponentStatus } from '../ComponentStatus/ComponentStatus';
+
+export const DemoPage = (componentPageProps: IDemoPageProps) => {
```

Comment:
Please kindly type as React.StatelessComponent :) The current declaration isn't equivalent from a typing POV.

Reply:
yes